### ISSUE TITLE
HM10 support?

### ISSUE THREAD

**Issue by marcbelmont** [giech/arduinotaskerbluetooth#4]
I'm using an HM-10 BLE/BT 4.0 module based upon TI's CC2540. It looks like your code does not support this type of module. I was able to find code for that module that does Arduino serial communication https://github.com/danasf/hm10-android-arduino . Any chance of supporting this module?

**Comment by giech** [comment]
Good idea. I have one lying around somewhere, though it will take a few weeks until I have the time to work on this again. I will update you once I have a version working to beta test it.

**Comment by giech** [comment]
There might be a bit of a further delay... I tried with a CC2541 module, but my Nexus 5X was not able to detect it (although my laptop could). There seems to be an issue with Oreo (e.g., [here](https://productforums.google.com/forum/#!msg/nexus/QxcbWBsnvWc/R0plFb5LDAAJ) and [here](http://www.electric-skateboard.builders/t/bluetooth-hm-10-not-discoverable-on-android-8-0-8-1-oreo/36713/56)), so it might be some time before I have a sane setup that can even minimally detect the module!

**Comment by marcbelmont** [comment]
Thanks for the update.
